src/Traits/HasPermissions.php

```diff
@@ -7,20 +7,44 @@
 trait HasPermissions
 {
     /**
-     * Grant the given permission to a role.
+     * Grant the given permission(s) to a role.
      *
-     * @param  $permission
+     * @param string|array|Permission|\Illuminate\Support\Collection $permissions
+     * @param array $params
      *
      * @return HasPermissions
      */
-    public function givePermissionTo($permission)
+    public function givePermissionTo($permissions, ...$params)
     {
```

Comment:
Could you make this work with method signature `givePermissionTo(...$permissions)`? I'm not sure why `params` is needed.